<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgLogin</class>
 <widget class="QDialog" name="DlgLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control Center Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/hub/cc_icon_resized.png</normaloff>:/hub/cc_icon_resized.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="5" column="0">
      <widget class="QPushButton" name="btn_resolve">
       <property name="text">
        <string>Resolve</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_4" columnstretch="2,0">
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_status">
         <property name="font">
          <font>
           <family>Linux Biolinum O</family>
          </font>
         </property>
         <property name="text">
          <string>lbl_status</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_ok">
         <property name="font">
          <font>
           <family>Linux Biolinum O</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">rgb(136, 138, 133)</string>
         </property>
         <property name="text">
          <string>Sign in</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_cancel">
         <property name="font">
          <font>
           <family>Linux Biolinum O</family>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lbl_login">
         <property name="font">
          <font>
           <family>Linux Biolinum O</family>
          </font>
         </property>
         <property name="text">
          <string>Bazaar username or email:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_login"/>
       </item>
       <item>
        <widget class="QLabel" name="lbl_password">
         <property name="font">
          <font>
           <family>Linux Biolinum O</family>
          </font>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_password">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="cb_save_credentials">
         <property name="font">
          <font>
           <family>Linux Biolinum O</family>
          </font>
         </property>
         <property name="text">
          <string>Remember me</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_register_link">
         <property name="font">
          <font>
           <family>Linux Biolinum O</family>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Register for Bazaar account</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
